android/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,9 @@
   <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
   <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
   <uses-permission android:name="android.permission.BLUETOOTH" />
+  <uses-permission android:name="android.permission.BLUETOOTH_SCAN" />
```

Comment:
Can we strictly assert here that we don't use this SCAN permission for location. Since we are using compileSdk 31, it should work.

Reply:
That'd be a useful change, thanks.

Reply:
**Impact:** This will remove some BLE Beacons the phone might scan, we might have to test this across a wide variety of phones to verify that no phones are left out due to manufacturer dependencies. I believe this should be a good change.

ref: https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#assert-never-for-location

**Other things:**

1. `ACCESS_FINE_LOCATION` is only required upto Android 11(API 30)
2. `ACCESS_COARSE_LOCATION` is only required upto Android 9(API 29)
3. For legacy Bluetooth permissions, restrict it to API level 30 (pt#4: [link](https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#declare-android12-or-higher))
These can be restricted by `maxSdk` property.

Reply:
I'd like to push `Other things` as a part of a separate ticket outside of the hotfix release branch

Reply:
Move **other things** to https://mosip.atlassian.net/browse/INJI-106 story